DependencyInjection/Configuration.php

```diff
@@ -298,6 +298,22 @@ private function createClientPluginNode()
                         ->end()
                     ->end()
                 ->end()
+                ->arrayNode('base_uri')
+                    ->canBeEnabled()
+                    ->addDefaultsIfNotSet()
+                    ->info('Set scheme, host and port in the request URI.')
+                    ->children()
+                        ->scalarNode('host')
```

Comment:
It should `uri` and not `host`, BaseUriPlugin can preprend a path, so someone should be able to use : `https://my-site/prepend-path`

Reply:
Yes, Also update here: https://github.com/php-http/HttplugBundle/pull/240/files#diff-18f3f26a96b967c0a5a3e13086d8367dR208